### PR TITLE
Initial Policy Reports integration - add to projects/namespaces, pods

### LINKED ISSUE #479: Extend Project/Namespaces view with Policy Reporter information

### LINKED ISSUE #480: Extend Namespace view with Policy Reporter information

### PR DESCRIPTION
**Pull request by jordojordo**
Fix #479 Fix #480 

This is the initial work for integration with PolicyReporter across the Dashboard.

- Adds Policy Reports Summary column to `Projects/Namespaces` list view
- Adds Policy Reports tab to Namespace and Pod detail views
- Bumped `@rancher/shell` to `0.3.25`

https://github.com/rancher/kubewarden-ui/assets/40806497/460f4e18-d0cc-48a7-8f0d-49d740f5afe7

### CODE REVIEW

File: pkg/kubewarden/l10n/en-us.yaml [hunk 1]
```diff
@@ -274,6 +274,25 @@ kubewarden:
     url:
       banner:
         unavailable: The Policy Reporter UI proxy URL is unavailble, please ensure that the UI is properly configured.
+    headers:
+      label: Policy Reports
```
Comment (viccuad) [inline pkg/kubewarden/l10n/en-us.yaml:278]:
```suggestion
      label: Compliance
```


Opening here a discussion about renaming the "Policy Reports" tab to something else ("compliance" as said in the past?)

Comment (jordojordo) [reply pkg/kubewarden/l10n/en-us.yaml]: I like the idea, I've changed the wording to "Compliance".

**Comment by kravciak** [comment]
I can't build the extension from this PR.

<details>
  <summary>Build log</summary>


```
~ VERSION=0.0.1 yarn build-pkg kubewarden
yarn run v1.22.19
$ ./node_modules/@rancher/shell/scripts/build-pkg.sh kubewarden
Building UI Package kubewarden
  Package name:    kubewarden-0.0.1
  Package version: 0.0.1
  Output formats:  umd-min
  Output directory: /home/kravciak/git/kw/ui/dist-pkg/kubewarden-0.0.1
~/git/kw/ui/pkg/kubewarden ~/git/kw/ui

⠙  Building for production as library (umd-min)...Starting type checking service...
Using 1 worker with 2048MB memory limit
⠦  Building for production as library (umd-min)...

 ERROR  Failed to compile with 2 errors                                                                  1:34:16 PM

 error  in /home/kravciak/git/kw/ui/pkg/kubewarden/index.ts

ERROR in /home/kravciak/git/kw/ui/pkg/kubewarden/index.ts(44,7):
44:7 Argument of type '{ path: { urlPath: string; endsWith: boolean; }[]; }' is not assignable to parameter of type 'string | LocationConfig'.
  Object literal may only specify known properties, and 'path' does not exist in type 'LocationConfig'.
    42 |   $plugin.addPanel(
    43 |     PanelLocation.RESOURCE_LIST,
  > 44 |     { path: [{ urlPath: 'explorer/projectsnamespaces', endsWith: true }] },
       |       ^
    45 |     { component: () => import('./components/PolicyReporter/ReporterPanel.vue') }
    46 |   );
    47 | 

 error  in /home/kravciak/git/kw/ui/pkg/kubewarden/index.ts

ERROR in /home/kravciak/git/kw/ui/pkg/kubewarden/index.ts(51,7):
51:7 Argument of type '{ path: { urlPath: string; endsWith: boolean; }[]; }' is not assignable to parameter of type 'string | LocationConfig'.
  Object literal may only specify known properties, and 'path' does not exist in type 'LocationConfig'.
    49 |   $plugin.addTableColumn(
    50 |     TableColumnLocation.RESOURCE,
  > 51 |     { path: [{ urlPath: 'explorer/projectsnamespaces', endsWith: true }] },
       |       ^
    52 |     {
    53 |       name:      'policy-reports',
    54 |       labelKey:  'kubewarden.policyReporter.headers.label',

Webpack Bundle Analyzer saved report to /home/kravciak/git/kw/ui/dist-pkg/kubewarden-0.0.1/report.html
 ERROR  Build failed with errors.
```
</details>

**Comment by jordojordo** [comment]
@kravciak You may need to reinstall the node modules, the `path` object was undefined previously.

**Comment by kravciak** [comment]
> @kravciak You may need to reinstall the node modules, the `path` object was undefined previously.

ah, right :) ...it helped, thanks!

**Review by viccuad — APPROVED** [review]
LGTM (for what is worth). It's looking great!

**Pull request merged**